client/src/components/Disclaimer.js

```diff
@@ -1,29 +1,45 @@
-import React, {Component} from 'react';
+import React, { Component } from 'react';
 import { ReactComponent as Logo } from '../img/Logo_CORONATRACKER_Logo.svg';
 import { ReactComponent as TextLogo } from '../img/Logo_CORONATRACKER_Text_Logo.svg';
 import Button from 'react-bootstrap/Button';
 
 class Disclaimer extends Component {
```

Comment:
I know we already have Disclaimer created, but could you rewrite this into a functional component in this PR for consistency?

Reply:
sure thing!